### PR TITLE
Integrete logs

### PR DESCRIPTION
**Pull request by chenshoo**

### CODE REVIEW

File: logging/ContextLogPropertiesWrapper.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {LogPropertiesWrapper, PolarisLogProperties} from "@enigmatis/polaris-logs";
+import {GraphQLLogProperties} from "./GraphQLLogProperties";
+
+export class ContextLogPropertiesWrapper implements LogPropertiesWrapper{
```
Comment (yarinvak) [inline logging/ContextLogPropertiesWrapper.ts]: determine if LogPropertiesWrapper is necessary if not remove it
if its needed i dont think that ``shouldLogRequestAndResponse`` should be here

File: logging/GraphQLLogger.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    error(PolarisLogProperties);
+    warn(PolarisLogProperties);
+    info(PolarisLogProperties);
+    trace(PolarisLogProperties);
+    debug(PolarisLogProperties);
+}
+
+@provide("InjectableLogger")
+export class GraphQLLogger extends PolarisLogger implements InjectableLogger {
+    constructor() {//applicationProperties: ApplicationLogProperties, loggerLevel: string, logstashHost: string, logstashPort: number) {
```
Comment (yarinvak) [inline logging/GraphQLLogger.ts]: what are the notes for

File: server/graphql.server.ts [hunk 3]
```diff
@@ -1,22 +1,63 @@
...
     private _polarisProperties: PolarisProperties;
+    private _logProperties: LogProperties;
+    private polarisLogger: InjectableLogger;
 
-    constructor(config: Config) {
-        let executableSchema = makeExecutableSchema(config.schema);
-        let propertiesPath = path.join('../', "properties.json");
-        PropertiesHolder.loadPropertiesFromFile(propertiesPath);
+    constructor(//@inject("InjectableLogger")polarisLogger :InjectableLogger,
+               // @inject("ISchemaCreator")creator :ISchemaCreator
```
Comment (yarinvak) [inline server/graphql.server.ts]: what are the notes for

File: server/graphql.server.ts [hunk 4]
```diff
@@ -1,22 +1,63 @@
...
+            let schema = creator.generateSchema()
+            let executableSchemaDefinition: { typeDefs: any, resolvers: any } = {
+                typeDefs: schema.def,
+                resolvers: schema.resolvers
+            };
+            let executableSchema = makeExecutableSchema(executableSchemaDefinition);*/
+        let config = { typeDefs:
+                [ 'schema {query: Query, mutation: Mutation}',
+                    'type Query {\n                    books: [Book] }',
+                    '\n             type Book implements CommonEntity {\n                 id: ID!\n                 creationDate: String,\n                 lastUpdateDate: String,\n                 dataVersion: Int!,\n                 title: String @upper,\n                 author: String,\n                 otherBook: Book\n             }\n         ',
```
Comment (yarinvak) [inline server/graphql.server.ts]: no no
there is a tag that allows you not to use `\n` and all this shit
it looks bad

and why at all we have a schema inside polaris??

File: server/graphql.server.ts [hunk 5]
```diff
@@ -33,11 +74,12 @@ export class PolarisGraphQLServer {
...
 
     public start() {
+        console.log("server start")
         let options = {};
         if (this._polarisProperties.port !== undefined) options['port'] = this._polarisProperties.port;
-
         app.listen(options, () => {
-            console.log(`🚀 Server ready at http://localhost:${options['port']}${this.server.graphqlPath}`);
+            let polarisProperties = new GraphQLLogProperties(`🚀 Server ready at http://localhost:${options['port']}${this.server.graphqlPath}`);
+            //this.polarisLogger.info(polarisProperties);
```
Comment (yarinvak) [inline server/graphql.server.ts]: do not use notes

File: logging/ContextLogPropertiesWrapper.ts (hunk 1 above)
Comment (chenshoo) [reply logging/ContextLogPropertiesWrapper.ts]: deleted for now

File: logging/GraphQLLogger.ts (hunk 2 above)
Comment (chenshoo) [reply logging/GraphQLLogger.ts]: not relevant, changed

File: server/graphql.server.ts (hunk 3 above)
Comment (chenshoo) [reply server/graphql.server.ts]: outdated

File: server/graphql.server.ts (hunk 4 above)
Comment (chenshoo) [reply server/graphql.server.ts]: outdated

File: server/graphql.server.ts (hunk 5 above)
Comment (chenshoo) [reply server/graphql.server.ts]: outdated

**Pull request closed without merging**